docs/_static/js/ga-tag-manager.js

```diff
@@ -0,0 +1,10 @@
+/**
+ *  google analytics script to add tracking
+ *  sphinx rtd theme uses deprecated version of tag script
+ */
+
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+
+gtag('config', 'UA-49099862-1');
```

Comment:
do they support using google tag manager?

Reply:
I mean should we refer to google tag container id instead of GA id?

Reply:
i was under the impression that tag manager only captures website data. updated. removed custom js, just extended the sphinx rtd theme layout so we input extra <head> and <body> elements

Reply:
![image](https://user-images.githubusercontent.com/54555734/75822996-dc42d100-5d98-11ea-8784-4f0c99269587.png)